arbnode/brotli_performance_test.go

```diff
@@ -0,0 +1,65 @@
+package arbnode
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/andybalholm/brotli"
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/util/testhelpers"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	BatchSizeLimit     = 20_000_000
+	CompressionLevel   = 11
+	RecompressionLevel = 11
+	NumMessages        = 100
+	MessageSize        = 1024 * 128
+)
+
+func TestGoLangBrotliPerformance(t *testing.T) {
```

Comment:
If you prefix the name with Benchmark* it will only run when running benchmarks. Some details about go benchmarking here: https://go.dev/blog/testing-b-loop